cli/src/utils/runPkgManagerInstall.ts

```diff
@@ -43,6 +43,11 @@ export const runPkgManagerInstall = async (
       } else {
         pkgJson.dependencies![pkgName] = `^${latestVersion.trim()}`; //eslint-disable-line @typescript-eslint/no-non-null-assertion
       }
+
+      // FIXME: temp fix for `next-auth@^4.11` requiring `next@12.2.5` as a peer dependency
+      if (pkg === "next-auth") {
```

Comment:
This only fixes noInstall mode. Need to fix regular mode too

Reply:
should be ok now, the fix i implemented is a bit ugly but hopefully it won't have to stay around for long

Reply:
Mind pushing your branch to our repo so we can test?